src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml

```diff
@@ -143,3 +143,11 @@ ra_location:
     requirements:
         raLocationId: ".+"
     condition: "request.headers.get('Accept') matches '/^application\\\\/json($|[;,])/'"
+
+institution_configuration_options_for_institution:
+    path:    /institution-configuration-options/{institution}
```

Comment:
no, those (options and ralocations) are separate concepts, that should be treated separately.

was more thinking along the lines of "queryability" - using Institution as a get parameter, as in searching. 

`\institution-configuration\{institution}\ra-location` and `\institution-configuration\{institution}\options` kinda makes sense, but both still require institution to be a resource, with ra-locations and options being a sub resource....

Nevermind, keep it as is, not worth the hassle :wink: